app/src/main/java/com/codesoom/assignment/product/application/ProductService.java

```diff
@@ -0,0 +1,55 @@
+package com.codesoom.assignment.product.application;
+
+import com.codesoom.assignment.product.domain.Product;
+import com.codesoom.assignment.product.infra.ProductRepository;
+import com.codesoom.assignment.product.ui.dto.ProductResponseDto;
+import com.codesoom.assignment.product.ui.dto.ProductSaveRequestDto;
+import com.codesoom.assignment.product.ui.dto.ProductUpdateRequestDto;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * 사용자 요청을 받아 상품 정보을 다룬다.
```

Comment:
이렇게 해도 무리 없을 것 같습니다.
```suggestion
 * 상품 정보을 다룬다.
```

Reply:
Service 주석을 어떻게 달아야할지 고민을 했는데 좋네요!